### PR TITLE
Makes dragon's blood turn you into a cooler lizard

### PR DESCRIPTION
**Pull request by Mqiib**
# Document the changes in your pull request

Adds a new subspecies of lizards: Draconids! They're just lizards with fire resistance, slight resistance to damage, and 2 more punch damage. They only come from the dragon's blood bottle from necropolis chests and are arguably still worse than just being a straight skeleton but hey it makes this roll actually give you something useful.

# Wiki Documentation

Perhaps some spooky cryptic mention of an ancient species of draconic humanoids on lavaland other than the askwalkers or something, I dunno.

# Changelog

:cl:  
rscadd: New rare species for powergaming lavalandists: Draconids!
tweak: Makes the lizard roll for dragon's blood bottles turn you into a draconid
experimental: WARNING: MINING BUFF
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
i hate lizards

**Comment by Mqiib** [comment]
> i hate lizards

same tbh

**Comment by Marmio64** [comment]
oh god, not going to lie, but miners really dont need extra armor when they can already get almost as much as xenobio. Also, i dont know about punishing miners with forced digitigrade legs. It definitely doesn't come out to being a powerful species or anything but i'm neutral on it.

**Comment by SomeguyManperson** [comment]
> Also, i dont know about punishing miners with forced digitigrade legs.

Lol

**Comment by Cartlord** [comment]
what if you turned ashies into these too

**Comment by cowbot92** [comment]
evil red lizard good

**Review by ToasterBiome — APPROVED** [review]
yeah ok

**Comment by Mqiib** [comment]
> oh god, not going to lie, but miners really dont need extra armor when they can already get almost as much as xenobio. Also, i dont know about punishing miners with forced digitigrade legs. It definitely doesn't come out to being a powerful species or anything but i'm neutral on it.

The original already forced digitigrade legs

image

The current version is literally just turning you into a red lizard with no benefits.

**Comment by Marmio64** [comment]
> > oh god, not going to lie, but miners really dont need extra armor when they can already get almost as much as xenobio. Also, i dont know about punishing miners with forced digitigrade legs. It definitely doesn't come out to being a powerful species or anything but i'm neutral on it.
> 
> The original already forced digitigrade legs
> 
> image
> 
> The current version is literally just turning you into a red lizard with no benefits.

honestly i'd just say remove the digitigrade legs from the original transformation and keep it in as a 33% chance to not get blessed with overpowered shit, and not get punished either. (Which you get along with some pretty good armor)

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
dont know how i like the inherent armor, punch is fine because of how little it will be used

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -60,6 +60,33 @@
...
+ Sadly they only get digitigrade legs. Can't have everything!
+*/
+/datum/species/lizard/draconid	
+	name = "Draconid"
+	id = "draconid"
+	limbs_id = "lizard"
+	fixed_mut_color = "A02720" 	//Deep red
+	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,DIGITIGRADE,HAS_FLESH,HAS_BONE)
+	inherent_traits = list(TRAIT_RESISTHEAT)	//Dragons like fire
+	burnmod = 0.8
```
Comment (ToasterBiome) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:76]:
```suggestion

```

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 2]
```diff
@@ -60,6 +60,33 @@
...
+	brutemod = 0.9 //something something dragon scales
```
Comment (ToasterBiome) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:77]:
```suggestion

```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
fuck i forgor the button to add them

**Comment by Smudged** [comment]
> dont know how i like the inherent armor, punch is fine because of how little it will be used

I don't think the extra armour is anything to worry about, considering it's a award from killing the ashdrake to turn you into a drachanoid anyways. 

Would rather keep all the changes and test them, The fire resistance is nice and flavourful.

**Comment by Mqiib** [comment]
> dont know how i like the inherent armor, punch is fine because of how little it will be used

The damage resistance is one of the main draws and is actually useful on lavaland if you're going around fighting bosses. Getting rid of it would basically make turning into a mini dragon person equivalent to just getting heat adaptation from genetics, which you can do every round. It would also once again make it basically objectively worse than the skeletonize option which _also_ makes you completely immune to all atmospherics, radiation, and disease for the low low price of looking spooky.

**Comment by ToasterBiome** [comment]
> > dont know how i like the inherent armor, punch is fine because of how little it will be used
> 
> The damage resistance is one of the main draws and is actually useful on lavaland if you're going around fighting bosses. Getting rid of it would basically make turning into a mini dragon person equivalent to just getting heat adaptation from genetics, which you can do every round. It would also once again make it basically objectively worse than the skeletonize option which _also_ makes you completely immune to all atmospherics, radiation, and disease for the low low price of looking spooky.

Yes but also go fight antag because armor resist

**Comment by Mqiib** [comment]
> > > dont know how i like the inherent armor, punch is fine because of how little it will be used
> > 
> > 
> > The damage resistance is one of the main draws and is actually useful on lavaland if you're going around fighting bosses. Getting rid of it would basically make turning into a mini dragon person equivalent to just getting heat adaptation from genetics, which you can do every round. It would also once again make it basically objectively worse than the skeletonize option which _also_ makes you completely immune to all atmospherics, radiation, and disease for the low low price of looking spooky.
> 
> Yes but also go fight antag because armor resist

This "valid the antags with it" argument can be made for almost every single piece of loot on lavaland, many of which don't even have a use in terms of actual mining and megafauna killing. Try again.

**Comment by ToasterBiome** [comment]
> > > > dont know how i like the inherent armor, punch is fine because of how little it will be used
> > > 
> > > 
> > > The damage resistance is one of the main draws and is actually useful on lavaland if you're going around fighting bosses. Getting rid of it would basically make turning into a mini dragon person equivalent to just getting heat adaptation from genetics, which you can do every round. It would also once again make it basically objectively worse than the skeletonize option which _also_ makes you completely immune to all atmospherics, radiation, and disease for the low low price of looking spooky.
> > 
> > 
> > Yes but also go fight antag because armor resist
> 
> This "valid the antags with it" argument can be made for almost every single piece of loot on lavaland, many of which don't even have a use in terms of actual mining and megafauna killing. Try again.

I do not believe this PR is worth maintaining from a code standpoint so unfortunately, I will be closing this now. Try again.

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**